cloudaeye/manifest.json

```diff
@@ -0,0 +1,53 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "4a0bb068-63fa-4cc2-a557-0452e8dca689",
+  "app_id": "cloudaeye",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "uninstallation": "README.md#Uninstallation",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Troubleshoot and resolve problems in seconds, remove toil, and earn customer trust.",
+    "title": "CloudAEye",
+    "media": [
+      {
+        "media_type": "image",
+        "caption": "Automated Root Cause Analysis",
+        "image_url": "images/root-cause-analysis.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "Notifications with Rich Context",
+        "image_url": "images/notifications.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "Centralized app analysis",
+        "image_url": "images/global-context.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "Assistant",
+        "image_url": "images/assistant.png"
+      }
+    ],
+    "classifier_tags": [
+      "Supported OS::Linux",
+      "Supported OS::macOS",
+      "Supported OS::Windows",
+      "Offering::Integration",
+      "Category::Cloud"
+    ]
```

Comment:
Add relevant tags from this list: https://docs.datadoghq.com/developers/integrations/check_references/#classifier-tags ex:
      "Queried Data Type::<FILL IN>", If pulling data from Datadog
      "Submitted Data Type::<FILL IN>" If sending in data to Datadog